src/Microsoft.Framework.Configuration.Binder/ConfigurationBinder.cs

```diff
@@ -12,28 +12,14 @@ namespace Microsoft.Framework.Configuration
 {
     public static class ConfigurationBinder
     {
-        public static TModel Bind<TModel>(IConfiguration configuration) where TModel : new()
+        public static void Bind(this IConfiguration configuration, object model)
```

Comment:
@HaoK Hao and I talked about this offline and we are going keep only the minimal building blocks for this. We can add sugar later based on customer feedback.